homeassistant/components/sunweg/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow for Sun WEG integration."""
+from sunweg.api import APIHelper
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_PLANT_ID, DOMAIN
+
+
+class SunWEGConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow class."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialise sun weg server flow."""
+        self.api: APIHelper = None
+        self.user_id = None
+        self.data: dict = {}
+
+    @callback
+    def _async_show_user_form(self, errors=None) -> FlowResult:
+        """Show the form to the user."""
+        data_schema = vol.Schema(
+            {
+                vol.Required(CONF_USERNAME): str,
+                vol.Required(CONF_PASSWORD): str,
+            }
+        )
+
+        return self.async_show_form(
```

Comment:
needs comment

Reply:
What do you mean? Isn't this an obvious line? I think that over-commenting is not needed.

Reply:
Yep sorry - my bad. I looked at it wrong. No comment needed here